main.c

```diff
@@ -254,30 +262,39 @@ void initAudio()
 	SDL_AudioSpec want;
 	SDL_AudioSpec have;
 
-	// init YM2151 emulation. 4 MHz clock
-	YM_Create(1.0f, 4000000);
-	YM_init(SAMPLERATE, 60);
+	/*
```

Comment:
can we print the list if -sound is passed without an argument? -keymap does something similar.